tooling/bundler/Cargo.toml

```diff
@@ -65,6 +65,7 @@ regex = "1"
 heck = "0.5"
 ar = "0.9.0"
 md5 = "0.7.0"
+rpm = "0.13.1"
```

Comment:
0.14 was released some time after the v2 pr, we might as well upgrade it here already.